buddyboss/buddyboss-platform
# #1834 Fixed Member/Profile types draft status

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1834
Fixes #1175

### How to test the changes in this Pull Request:

1. Add group type
2. Change status to draft
3. Add/Edit group and try to add/change group type
4. See that draft type/s are still available as option

Expected behavior
Draft types should be not selectable

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
